calico_containers/pycalico/netns.py

```diff
@@ -342,7 +342,7 @@ def check_output(self, command):
         """
         command = self._get_nets_command(command)
         _log.debug("Run command: %s", command)
```

Comment:
Please let's add a comment here:

    # Note: we must not also capture the stderr here (as with
    # stderr=STDOUT), because an otherwise successful 'ip netns exec <ns>
    # <command> ...' call will write error messages to stderr if there are
    # any bogus namespaces on the system.  See
    # https://github.com/projectcalico/libcalico/issues/148 for detailed
    # discussion of this.